lib/msf/core/modules/metadata/cache.rb

```diff
@@ -35,10 +35,10 @@ def refresh_metadata_instance(module_instance)
   #  Returns the module data cache, but first ensures all the metadata is loaded
   #
   def get_metadata
-    @mutex.synchronize {
+   @mutex.synchronize {
```

Comment:
Minor: Why was indent reduced by one here and few lines below?

Reply:
its was off previously